libcentrifugo/metrics.go

```diff
@@ -120,7 +101,7 @@ func (c *metricCounter) LoadRaw() int64 {
 }
 
 func (c *metricCounter) LastIn() int64 {
-	return atomic.LoadInt64(&c.value)
+	return atomic.LoadInt64(&c.lastIntervalDelta)
```

Comment:
Why atomic load? Before all reads and writes to delta were done only under lock. But I may be missing something on my phone!